### PR TITLE
Various cleanups.

### PR DESCRIPTION
**Pull request by davidlehn**
***
<a href="https://pr-preview.s3.amazonaws.com/davidlehn/did-core/pull/633.html" title="Last updated on Feb 26, 2021, 5:41 PM UTC (d3114f1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/633/ed12dd5...davidlehn:d3114f1.html" title="Last updated on Feb 26, 2021, 5:41 PM UTC (d3114f1)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4178,7 +4177,7 @@ <h2>DID URL Dereferencing</h2>
         <dd>
 If the <code>dereferencing</code> function was called and successful, this MUST
 contain a <a>resource</a> corresponding to the <a>DID URL</a>. The
-<code>contentStream</code> MAY be a <a>resource</a> such as a <a href="">DID
+<code>contentStream</code> MAY be a <a>resource</a> such as a <a>DID
 Document</a> that is serializable in one of the conformant
```
Comment (rhiaro) [inline index.html:4181]:
```suggestion
document</a> that is serializable in one of the conformant
```

**Review by rhiaro — APPROVED** [review]

**Comment by msporny** [comment]
@davidlehn -- due to the massive editorial pass that's happening on the specification right now (expected to be done close to the end of this week), this PR is bound to have/create lots of merge conflicts. I'm going to hold off on merging it until the editorial pass has been completed.

**Comment by msporny** [comment]
@davidlehn -- ok, NOW is the right time to make this pass. Can you please make it again... don't know if this PR is salvageable. Please close it if it isn't.

**Comment by peacekeeper** [comment]
I also thought it would be best to do these tiny spelling and formatting fixes at the very end to avoid merge conflicts. But having said that, thanks a lot for spending the time to read through everything in such detail!

**Comment by msporny** [comment]
@davidlehn -- I have to make a snapshot of the spec for the transition to Candidate Recommendation, and all PRs must be processed by that time. If I don't hear back from you in 48 hours, I'm going to close this PR due to the merge conflicts. Please respond.

**Comment by davidlehn** [comment]
@msporny Rebased.  That was a bit painful!  I think all the fixes got through.  Found a few more with all the new updates.

@peacekeeper Yeah, I had some poor timing with this PR.  Oops.  Mostly just use vim spellcheck (with some handholding), other vim markup checking, vim grammar(ish) checker, tidy, and some visual indent wrangling. Doesn't take all that long to do a pass every so often.  Took longer to fix the rebase than to do the original edits!

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**